skills_DS/react/src/pages/admin/GetSkills.js

```diff
@@ -15,12 +15,15 @@ const GetSkills = props => {
   const dismissAlert = () => {
     setAlert({ visible: false })
   }
+  
+  const REACT_APP_GOOGLE_MAPS_KEY = "AIzaSyDXcZrBQloC9mdrUbmKrLo9Q8AqfI1ifs8";
```

Comment:
Is this safe?

Reply:
I forgot to mention, due to how webpack is setup we are unable to use env variables in React. This is still safe since the Google map key is meant to be public anyway.